templates/docs/base/code.md

```diff
@@ -60,6 +60,14 @@ View example of the code snippet
 View example of the code snippet
 </a></div>
 
+**Code snippet with dropdown**
```

Comment:
Now, that these are not  really 'dropdowns', does it still make sense to call them like this?

I wouldn't call them 'selects' (as that's HTML jargon), but what are they by function? Dropdowns? Menus? How does the spec call them?

Reply:
[Here](https://discourse.ubuntu.com/t/code-snippet-pattern/19195) they're called "version switchers", in [the design](https://app.zeplin.io/project/5fa270d8eee4dbb267e579f1/screen/5ffc2fdcb65e6a99b78f9d47) they're called "dropdowns".

I did initially go with just calling them selects, but it felt inappropriate and like it was tied to a form. "Version switcher" won't suit every use case, but we could also go with "switcher", it's ultimately what the element does. That said, I think they can also be considered as "dropdowns", as it's _also_ what the element does. "Dropdown switcher"?